ui/component/layoutWrapperText/view.jsx

```diff
@@ -46,25 +46,22 @@ function LayoutWrapperText(props: Props) {
           <h1 className="media__title-text">{title}</h1>
         </div>
 
-        <div className="media__subtitle--between">
-          <DateTime uri={uri} show={DateTime.SHOW_DATE} />
```

Comment:
I think at least the date should stay so you know if what you are reading is recent/old. We added the view count too, just because we had the extra space from the adding the date.